### DISCUSSION TITLE
findOne($query) ?

### DISCUSSION THREAD

**Discussion by 0utc0ntr0l2021** [jenssegers/laravel-mongodb#2478] [Q&A] [answered]
how to make this code work ?

		$query = array('endip' => array('$gte' => (float)$ipnum));
		$range = RangeModel::findOne($query);

**Comment by Giacomo92** [comment, marked as answer]
Hi @0utc0ntr0l2021 ,
try this:
`$range = RangeModel::where('endip', '>=', (float)$ipnum)->first();`
